esdoc-publish-html-plugin/src/Builder/DocBuilder.js

```diff
@@ -676,9 +676,15 @@ export default class DocBuilder {
         .replace(/<.*?>/g, (a)=> a.replace(/,/g, '\\Z'))
         .replace(/{.*?}/g, (a)=> a.replace(/,/g, '\\Z').replace(/:/g, '\\Y'));
       const innerTypes = inner.split(',').map((v)=>{
-        const tmp = v.split(':').map((v)=> v.trim());
-        const paramName = tmp[0];
-        let typeName = tmp[1].replace(/\\Z/g, ',').replace(/\\Y/g, ':');
+        let tmp, paramName, typeName = '';
+        try {
+          tmp = v.split(':').map((v)=> v.trim());
+          paramName = tmp[0];
```

Comment:
Instead of the whole try/catch, why not simply use `const paramName = "tmp[0]";` ?

Reply:
Not to mention the fact that you remove the `const` attribute from 2 vars by your "fix".